### ISSUE TITLE
website download button

### ISSUE THREAD

**Issue by leandroruel** [at-import/breakpoint#119]
I took a long time to discover that button at top of site is to download this project, it would be better to put a link or another button with the text "download" on the site, that github logo is almost hidden

**Comment by TomasDuda** [comment]
I agree, there should be a normal download/go to repo button instead of the weird icon.

**Comment by Snugug** [comment]
Website needs an overhaul, @codingdesigner?
